ldk/node/src/hostClients/keyboardClient.ts

```diff
@@ -102,22 +102,27 @@ export default class KeyboardClient
     hotKeys: HotKeyRequest,
     listener: StreamListener<HotKeyEvent>,
   ): StoppableStream<HotKeyEvent> {
-    const message = generateHotkeyStreamRequest(hotKeys);
+    const message = generateHotkeyStreamRequest(hotKeys).setSession(
+      this.createSessionMessage(),
+    );
     return new TransformingStream(
       this.client.keyboardHotkeyStream(message),
       transformHotKeyEvent,
       listener,
     );
   }
 
-  streamText(): StoppableStream<string> {
+  streamText(listener: StreamListener<string>): StoppableStream<string> {
```

Comment:
This was not taking a listener. I'll triple check this with the other LDKs, but I'm 90% certain this should take a callback.

Reply:
You are 100% correct! The stoppable streams absolutely should accept a listener.